quadrat/sass/templates/_meta.scss

```diff
@@ -1,37 +1,61 @@
 .post-meta {
 	align-items: center;
 	justify-content: center;
+
+	.wp-block-group & {
+		// 1px needed to account for the separator
+		gap: calc( var(--wp--custom--gap--baseline) + 1px);
+	}
+
 	@include break-small-only() {
 		padding-top: 0 !important;
 		margin-bottom: -20px;
 	}
 
-	.wp-block-post-date::before {
-		display: none;
-	}
+	> * {
+		&:not(:first-child):before {
+			align-items: center;
+			background: transparent;
+			color: var(--wp--custom--color--foreground);
+			content: "·";
+			display: flex;
+			margin-right: 0;
+			-webkit-mask-image: none;
+			mask-image: none;
+		}
 
-	> *,
-	.wp-block-post-date {
-		&::before {
-			content: "";
+		&:first-child:before {
+			background: transparent;
+			-webkit-mask-image: none;
+			mask-image: none;
 		}
-	}
 
-	.wp-block-post-terms {
-		&::before {
+		> * {
 			color: var(--wp--custom--color--foreground);
-			content: "·";
-			margin-right: 8px;
+			font-size: var(--wp--preset--font-size--tiny);
 		}
+	}
 
-		// Needed while https://github.com/WordPress/gutenberg/issues/31710 is sorted.
-		color: transparent;
-		a {
-			color: var(--wp--custom--color--foreground);
-			&:not(:last-child):after {
-				// to be removed after https://github.com/WordPress/gutenberg/pull/32812 lands
-				content: ",";
-			}
+	.wp-block-post-author__content {
+		align-items: center;
```

Comment:
I don't think we need `align-items: center` here, it looks like `justify-content` is enough. It also seems to make the author content horizontally centered:

![Screenshot 2021-09-23 at 11 10 37](https://user-images.githubusercontent.com/1645628/134490400-31cdffea-7a6b-4400-990a-608487162c9b.png)

Reply:
Thanks, done!